gemrb/GUIScripts/pst/NewLife.py

```diff
@@ -250,9 +250,10 @@ def AcceptPress():
 	GemRB.SetPlayerStat(1, IE_CHR, Stats[5])
 
 	if LevelUp:
-		# hp is handled in GUIREC
+		# Return to the RecordsWindow
+		NewLifeWindow.Close()
 		import GUIREC
-		GUIREC.OpenLevelUpWindow ()
+		GUIREC.UpdateRecordsWindow(GUIREC.RecordsWindow)
```

Comment:
this too _should_ be useless, and if it isn't then there is a bug. 

`UpdateRecordsWindow` is already bound to the `ACTION_WINDOW_FOCUS_GAINED` event for the records window so it should be getting called automatically anytime the records window gains focus.

Reply:
I tried first just with GUIREC.RecordsWindow.Focus() , but for some reason that wasn't working. But I'll try and figure out why, if that is how it should be done.

Reply:
I think you misunderstand me. You shouldn't have to do anything. when `NewLifeWindow` closes the records window will automatically gain focus and automatically call `UpdateRecordsWindow`

Reply:
The records window opens the level up sub window, but looking at AcceptLevelUp I think I might know where it is going wrong, just need to try something...

Reply:
I'm not sure why you are scratching your head. I think the problem is that you are closing the window at the top of the function instead of the bottom.

Reply:
Doh, that's exactly it. I just assumed that the function was being called by closing the NewLife window...